package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "react-smart-tabs",
-  "version": "0.1.0",
+  "version": "0.2.5-alpha",
```

Comment:
Why this jump?

Reply:
weren't you guys using `npm version` on master for publishing? :man_facepalming:

Reply:
anyway, created https://github.com/EvoluxBR/react-smart-tabs/issues/45 for dealing with this later

Reply:
the thing here is I tested  4 "failed alpha releases" attempting to get types coming inside the npm package (and proper set up)